test/utils/context.ts

```diff
@@ -27,13 +28,17 @@ export type Context = {
         USDC: IERC20
         USDT: IERC20
         WETH: IERC20
+        DAIForArbitrumBridge: IERC20
+        USDCForArbitrumBridge: IERC20
+        USDTForArbitrumBridge: IERC20
```

Comment:
Maybe we need to come up a new way to organize the addresses since it starts to diversify between Mainnet and Arbitrum. We could think more and do the refinement in another PR.